Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/) for Kodi add-ons: video/plugin.video.nasa.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-video/-/plugin-video-nasa/horizontal-auto.svg)

gade01 reviewed (commented):
Looks very good to me.

- Corrections are added
- New languages
- Translations using Weblate translation memory.

weblate committed:
Translated using Weblate (Ossetian (os_os))

Currently translated at 16.6% (1 of 6 strings)

Translated using Weblate (Tajik (tg_tj))

Currently translated at 50.0% (3 of 6 strings)

Translated using Weblate (Telugu (India) (te_in))

Currently translated at 16.6% (1 of 6 strings)

Translated using Weblate (Silesian)

Currently translated at 50.0% (3 of 6 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 33.3% (2 of 6 strings)

Translated using Weblate (Maori)

Currently translated at 33.3% (2 of 6 strings)

Translated using Weblate (Kannada (India) (kn_in))

Currently translated at 16.6% (1 of 6 strings)

Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 33.3% (2 of 6 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 66.6% (4 of 6 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Added translation using Weblate (Ossetian (os_os))

Added translation using Weblate (Tajik (tg_tj))

Added translation using Weblate (Telugu (India) (te_in))

Added translation using Weblate (Silesian)

Added translation using Weblate (Sinhala (Sri Lanka) (si_lk))

Added translation using Weblate (Sicilian)

Added translation using Weblate (Moldavian (Romania) (ro_md))

Added translation using Weblate (Maori)

Added translation using Weblate (Kannada (India) (kn_in))

Added translation using Weblate (Asturian (Spain) (ast_es))

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/ast_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/kn_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/mi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/os_os/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/si_lk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/szl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/te_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-nasa/tg_tj/
Translation: Kodi add-ons: video/plugin.video.nasa
